src/analysis/vector/geometry_checker/qgsgeometrymissingvertexcheck.cpp

```diff
@@ -73,11 +76,32 @@ void QgsGeometryMissingVertexCheck::fixError( const QMap<QString, QgsFeaturePool
   {
     error->setFixed( method );
   }
+  if ( method == AddMissingVertex )
```

Comment:
Would a switch be more appropriate here? (for future build warnings)

Reply:
it's an integer that's passed in because the API is from the base class where the enum is unknown.
Would you cast to the proper enum for the switch?

Reply:
@m-kuhn done here https://github.com/3nids/QGIS/commit/dc6a87c14ddf51ea3bdb41a47b674b3fff1bf326#diff-5893c5403c3be6af7ddfbf56586c08f6R139